app/views/likes/_like.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  <% if like = current_user.liked(bookmark) %>
+    <%= link_to [bookmark, like], class: 'btn btn-danger', method: :delete do %>
+      <i class="glyphicon glyphicon-star-empty"> </i>&nbsp; Unlike
+    <% end %>
+  <% else %>
+    <%= link_to [bookmark, like.new], class: 'btn btn-primary', method: :post do %>
```

Comment:
@LuisMenjivar Does this line work OK? I'd expect an error to be raised at `like.new`